logger/drain/drain.go

```diff
@@ -0,0 +1,59 @@
+package drain
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"net/url"
+	"os"
+
+	"github.com/coreos/go-etcd/etcd"
+)
+
+func GetDrain() string {
+	host := getopt("HOST", "127.0.0.1")
+
+	etcdPort := getopt("ETCD_PORT", "4001")
+	etcdPath := getopt("ETCD_PATH", "/deis/logs")
+
+	client := etcd.NewClient([]string{"http://" + host + ":" + etcdPort})
+
+	s, err := client.Get(etcdPath+"/drain", true, false)
+	if err != nil {
+		return ""
```

Comment:
How would you prefer this done?
etcd gives out an error `100: Key not found (/deis/logs/drain) [2674]`